example/bin/ci/example.sh

```diff
@@ -10,13 +10,13 @@
 #     Provides an example of customizing an ORCA build by adding or
 #     modifying jobs. See real working examples:
 #
-#     - https://github.com/acquia/coding-standards-php/blob/v0.5.0/.travis.yml#L52 and
+#     - https://github.com/acquia/coding-standards-php/blob/develop/.github/workflows/orca.yml#L97 and
```

Comment:
When linking to files on GitHub--_especially_ when pointing to a specific line number--always point to a tagged version, not a branch. Because the branch will change over time and may change or even delete the referenced file, resulting in broken links. Whereas the tag, by definition, will always be what it was when you linked to it.

Reply:
Noted

Reply:
Looks like this didn't get changed--the links still points to the develop branch instead of the latest tag.